charts/datadog/templates/system-probe-configmap.yaml

```diff
@@ -53,6 +53,8 @@ data:
         enabled: {{ $.Values.datadog.securityAgent.runtime.syscallMonitor.enabled }}
       network:
         enabled: {{ $.Values.datadog.securityAgent.runtime.network.enabled }}
+      remote_configuration:
+        enabled: {{ $.Values.datadog.securityAgent.runtime.remoteConfiguration.enabled }}
```

Comment:
The config option in this file and `DD_REMOTE_CONFIGURATION_ENABLED` are unrelated. This option is only used to tell the system-probe container to receive policy updates through remote config.
`DD_REMOTE_CONFIGURATION_ENABLED` isn't used directly because we don't want system-probe to use remote config by default, even if it's enabled at the core-agent level.